lib/src/utils.dart

```diff
@@ -122,6 +122,20 @@ Uri packageToAssetUrl(Uri url) => url.scheme == 'package'
           ..addAll(url.pathSegments.skip(1)))
     : url;
 
+/// Returns a `asset:` URL converted to a `package:` URL.
```

Comment:
Where is the `asset:` uri logic defined? For barback?

Might be nice to clarify this...